### PR TITLE
fix: remove redundant name attribute from logging plugins

### PR DESCRIPTION
**Pull request by kayx23**
### Description

Unique ID for batch processor (attribute `name`) are hardcoded in for quite many logging plugins. The docs, schema of certain plugins, and tests, previously treat `name` as a configurable attribute, which could create confusions, such as https://github.com/apache/apisix/issues/10474.

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by shreemaan-abhishek — APPROVED** [review]

**Comment by kayx23** [comment]
Lmk if I should apply this change for other plugins in the same PR. Quite a few logging plugins have the same issues if I'm not mistaken.

**Comment by moonming** [comment]
> Lmk if I should apply this change for other plugins in the same PR. Quite a few logging plugins have the same issues if I'm not mistaken.

yes, please do it

**Review by pottekkat — DISMISSED** [review]
LGTM and +1 to go ahead and fix this in other plugins as well.

**Comment by kayx23** [comment]
Just saw in `clickhouse-logger` the name is indeed configurable:

https://github.com/apache/apisix/blob/d8bd34414239ed36942c19f435ac10ad0d5e57c0/apisix/plugins/clickhouse-logger.lua#L28-L41

Are we going to:

1. make the other logger plugin names configurable as well; or 
2. Just update the docs for those that are not configurable?

**Review by monkeyDluffy6017 — DISMISSED** [review]

**Comment by moonming** [comment]
> Just saw in `clickhouse-logger` and `sys-logger` the namea are indeed configurable:
> 
> https://github.com/apache/apisix/blob/d8bd34414239ed36942c19f435ac10ad0d5e57c0/apisix/plugins/clickhouse-logger.lua#L28-L41
> 
> image
> 
> Are we going to:
> 
> 1. make the other logger plugin names configurable as well;
> 2. update `clickhouse logger` and `sys logger` to be consistent with others, which hard code the name; or
> 3. Just update the docs for those that are not configurable (done)?

**Comment by kayx23** [comment]
Ok as it turns out (thanks to the failed CI tests) that this `name` attribute is actually configurable and its value gets exported to prometheus to identify batch processors.

The reason why the `name` attribute was documented to each of these logging plugin, despite being a batch processor attribute, was likely because these loggers initiate batch processor instances with specific names (the hard coded values), so they are the default values specific to these plugins.

In short, these docs should not be removed and the documented default values are consistent with what goes in the code.
